include/subhalo.h

```diff
@@ -206,6 +206,8 @@ class Subhalo : public Identifiable<subhalo_id_t>, public Spatial<float> {
 	float Mvir = 0;
 	/// gas mass in the subhalo [Msun/h]. This is different than 0 if the input simulation is a hydrodynamical simulation.
 	float Mgas = 0;
+	/// numver of dark matter particles in the halo
```

Comment:
num*b*er

Reply:
Would you like me to commit a change to update this?

Reply:
If not a problem; otherwise it's fine, I can also fix it later